doc/install.txt

```diff
@@ -4,891 +4,17 @@
 Installing Theano
 =================
 
+On the following pages you will find different installation instructions depending
+on your needs. 
 
-Requirements
-------------
+- :ref:`install_ubuntu`
+- :ref:`install_centos6`
+- :ref:`install_macos`
+- :ref:`install_windows`
+- :ref:`install_generic`
```

Comment:
I don't think we need a generic installation section. When are you installing something generically?
And see some people getting confused by "generic" and "others" installs.